lib/internal/socket_list.js

```diff
@@ -2,7 +2,7 @@
 
 const EventEmitter = require('events');
 
-// This object keep track of the socket there are sended
+// This object keeps track of the socket that is sent
```

Comment:
I'm not 100% sure, but I think it should be 'sockets' instead of 'socket' and then s/is/are/

Also, the `SocketListReceive` description below needs to be updated similarly as well.